core/src/test/java/io/el/concurrent/DefaultEventLoopGroupTest.java

```diff
@@ -0,0 +1,279 @@
+package io.el.concurrent;
+
+import static io.el.internal.ObjectUtil.checkNotNull;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.ThreadFactory;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.concurrent.atomic.AtomicReferenceFieldUpdater;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+@SuppressWarnings("unchecked")
+public class DefaultEventLoopGroupTest {
+
+  private static final SingleThreadEventLoop eventLoop = new MockEventLoop(
+      new ThreadPerTaskExecutor(Executors.defaultThreadFactory())
+  );
```

Comment:
It would be better just `Mockito.mock(SingleThreadEventLoop.class)` instead of `MockEventLoop`.

And I think `MockEventLoop` class you created is not needed

Reply:
done in oral discussion.
decided to delete DefaultEventLoopGroupTest.java